eng/pipelines/templates/jobs/smoke.tests.yml

```diff
@@ -12,6 +12,9 @@ jobs:
   - ${{ if eq(parameters.Daily, false) }}:
     - job: smoke_test_eligibility
       displayName: Check Smoke Test Eligibility
+      pool:
```

Comment:
Where was this set before? Did it just use default value?

Reply:
Yep, I think it was just defaulting to ubuntu/azure pipelines.